WebdriverIO version update

muttavarapu opened the pull request:
#### Asana Task
[asana](https://app.asana.com/0/15335422358307/661122851918758/f)
# Background:
We use Selenium and webdriver.io to do automated testing. 
# Problem:
webdriver.io is out of date and this is causing issues making new tasks and fixing existing ones. 
# Solution:
Update webdriver.io
# Touch Points:
https://github.com/MRN-Code/coins-browser-test
All testing should follow protections in Bug and Feature Test Cases

mstone121 commented:
Here's a full list;

```
1) shell login passes cookie with successful request:
expected Object {
  date: 'Fri, 25 May 2018 18:31:11 GMT',
  server: 'Apache/2.4.7 (Ubuntu)',
  'x-powered-by': 'PHP/5.5.9-1ubuntu4.25',
  'cache-control': 'max-age=0',
  expires: 'Fri, 25 May 2018 18:31:11 GMT',
  'content-length': '29',
  connection: 'close',
  'content-type': 'text/html'
} to have property 'set-cookie'
running chrome
AssertionError: expected Object {
  date: 'Fri, 25 May 2018 18:31:11 GMT',
  server: 'Apache/2.4.7 (Ubuntu)',
  'x-powered-by': 'PHP/5.5.9-1ubuntu4.25',
  'cache-control': 'max-age=0',
  expires: 'Fri, 25 May 2018 18:31:11 GMT',
  'content-length': '29',
  connection: 'close',
  'content-type': 'text/html'
} to have property 'set-cookie'
    at Assertion.fail (/Users/mstone/stuff/coins-browser-test/node_modules/should/cjs/should.js:275:17)
    at Assertion.value [as property] (/Users/mstone/stuff/coins-browser-test/node_modules/should/cjs/should.js:356:19)
    at Context.it (/Users/mstone/stuff/coins-browser-test/test/shell-login.js:64:30)
    at Promise.F (/Users/mstone/stuff/coins-browser-test/node_modules/core-js/library/modules/_export.js:36:28)

2) study view subjects pages should be able to view subject details:
element not visible
running chrome
Error: An element command could not be completed because the element is not visible on the page.
    at Object.me.view.subjectDetails.ursi [as subjectDetails] (test/lib/study.js:40:6)
    at Context.it (test/study.js:33:18)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elementIdClick("0.39308669322640744-9") - click.js:20:22

3) study view subjects pages should be able to add a global subject tag:
An element could not be located on the page using the given search parameters.
running chrome
Error: An element could not be located on the page using the given search parameters.
    at Context.it (test/study.js:39:10)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at element("=Subject Tags") - click.js:12:17
    at click("=Subject Tags") - at Function.me.view.subjectDetails.addTag (test/lib/study.js:45:8)

4) study view subjects pages should be able to edit a global subject tag:
An element could not be located on the page using the given search parameters.
running chrome
Error: An element could not be located on the page using the given search parameters.
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at element("//*[contains(text(), "testTag_1527273163026")]//..//form//input[@value="Edit"]") - click.js:12:17
    at click("//*[contains(text(), "testTag_1527273163026")]//..//form//input[@value="Edit"]") - at Context.it (test/study.js:52:12)

5) Add subject tags should save tag edits:
element ("#subject_tags_table tbody") still without text after 1000ms
running chrome
Error: element ("#subject_tags_table tbody") still without text after 1000ms
    at Context.it (test/subject-tags.js:116:8)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elementIdText("0.6343580138201692-26") - getText.js:35:50

6) Add subject tags should remove deleted tags:
An element could not be located on the page using the given search parameters ("#subject_tags_table tbody").
running chrome
Error: An element could not be located on the page using the given search parameters ("#subject_tags_table tbody").
    at Array.forEach (native)
    at Context.it (test/subject-tags.js:126:16)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("#subject_tags_table tbody") - getText.js:18:17
    at getText("#subject_tags_table tbody") - at sampleTags.forEach (test/subject-tags.js:139:27)

7) subject enroll handle new subject matches form should be enroll with existing subject:
An element could not be located on the page using the given search parameters.
running chrome
Error: An element could not be located on the page using the given search parameters.
    at Context.it (test/subject-enroll.js:43:19)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at element("input[type=button][value=Continue]") - click.js:12:17
    at click("input[type=button][value=Continue]") - at Object.me.new.handleSubjectMatchesExisting (test/lib/subject.js:146:12)

8) navigate to asmt should change asmt study:
element not visible: Element is not currently visible and may not be manipulated
running chrome
running chrome
Error: An element command could not be completed because the element is not visible on the page.
    at Object.me.selectAsmtStudy (test/lib/nav/navigation.js:54:8)
    at Context.it (test/navigateToAsmt.js:23:9)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elementIdClick("0.9157013253415742-4") - selectByAttribute.js:46:22

9) Perform various imports and verify that they function correctly Import instruments and assessments Should go to asmt and select NITEST (study_id 2319):
element not visible: Element is not currently visible and may not be manipulated
running chrome
Error: An element command could not be completed because the element is not visible on the page.
    at Object.me.selectAsmtStudy (test/lib/nav/navigation.js:54:8)
    at Context.it (test/fileImport.js:51:11)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elementIdClick("0.10117279303228299-13") - selectByAttribute.js:46:22

10) Perform various imports and verify that they function correctly Import instruments and assessments Should see the upload is successful:
expected 'Import Instrument From CSV:\nQuestion Prefix:\n\nGenerate Template   Browse for file' to contain 'Instrument successfully added'
running chrome
AssertionError: expected 'Import Instrument From CSV:\nQuestion Prefix:\n\nGenerate Template   Browse for file' to contain 'Instrument successfully added'
    at Assertion.fail (/Users/mstone/stuff/coins-browser-test/node_modules/should/cjs/should.js:275:17)
    at Assertion.value [as containEql] (/Users/mstone/stuff/coins-browser-test/node_modules/should/cjs/should.js:356:19)
    at Context.it (/Users/mstone/stuff/coins-browser-test/test/fileImport.js:70:52)
    at Promise.F (/Users/mstone/stuff/coins-browser-test/node_modules/core-js/library/modules/_export.js:36:28)

11) Perform various imports and verify that they function correctly Import instruments and assessments Should upload assessments:
An element could not be located on the page using the given search parameters.
running chrome
Error: An element could not be located on the page using the given search parameters.
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at element("select#instrument_id") - selectByAttribute.js:22:17
    at selectByAttribute("select#instrument_id", "value", 24063) - selectByValue.js:40:17
    at selectByValue("select#instrument_id", 24063) - at Context.it (test/fileImport.js:79:15)

12) Perform various imports and verify that they function correctly Import instruments and assessments Should see the upload is successful:
An element could not be located on the page using the given search parameters ("div#new_asmts_info").
running chrome
Error: An element could not be located on the page using the given search parameters ("div#new_asmts_info").
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("div#new_asmts_info") - getText.js:18:17
    at getText("div#new_asmts_info") - at Context.it (test/fileImport.js:96:15)

13) instrument should create a new instrument:
An element could not be located on the page using the given search parameters ("input[name=label]").
running chrome
Error: An element could not be located on the page using the given search parameters ("input[name=label]").
    at Context.it (test/instrument.js:67:16)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("input[name=label]") - setValue.js:29:17
    at setValue("input[name=label]", "selenium test instrument label") - at Object.me.create.options [as create] (test/lib/instrument.js:83:6)

14) instrument should load the new instrument:
An element could not be located on the page using the given search parameters ("input[name=label]").
running chrome
Error: An element could not be located on the page using the given search parameters ("input[name=label]").
    at Context.it (test/instrument.js:74:16)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("input[name=label]") - getValue.js:18:17
    at getValue("input[name=label]") - at Object.me.fromHtml (test/lib/instrument.js:107:28)

15) instrument should allow the new instrument to be edited:
An element could not be located on the page using the given search parameters ("[name=label]").
running chrome
Error: An element could not be located on the page using the given search parameters ("[name=label]").
    at node_modules/lodash/lodash.js:4925:15
    at baseForOwn (node_modules/lodash/lodash.js:3010:24)
    at node_modules/lodash/lodash.js:4894:18
    at Function.forEach (node_modules/lodash/lodash.js:9342:14)
    at Object.me.edit (test/lib/instrument.js:94:7)
    at Context.it (test/instrument.js:78:16)
    at elements("[name=label]") - setValue.js:29:17
    at setValue("[name=label]", "selenium test instrument label edit") - at _.forEach (test/lib/instrument.js:96:27)

16) instrument should load the edited instrument properties:
An element could not be located on the page using the given search parameters ("input[name=label]").
running chrome
Error: An element could not be located on the page using the given search parameters ("input[name=label]").
    at Context.it (test/instrument.js:85:16)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("input[name=label]") - getValue.js:18:17
    at getValue("input[name=label]") - at Object.me.fromHtml (test/lib/instrument.js:107:28)

17) instrument should print the instrument_id in a hidden field:
An element could not be located on the page using the given search parameters ("[name=instrument_id]").
running chrome
Error: An element could not be located on the page using the given search parameters ("[name=instrument_id]").
    at Context.it (test/instrument.js:88:16)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("[name=instrument_id]") - getValue.js:18:17
    at getValue("[name=instrument_id]") - at Object.me.setInstrumentIdFromPage (test/lib/instrument.js:42:24)

18) instrument should locate the newly created instrument:
number or type of arguments don't agree with setValue command
running chrome
Error: number or type of arguments don't agree with setValue command
    at Context.it (test/instrument.js:94:16)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at setValue("#instrument_grid_filter input[type=search]", ) - at Object.me.filterList (test/lib/instrument.js:37:19)

19) instrument should unlock the instrument:
An element could not be located on the page using the given search parameters ("[data-instrument_id="undefined"] input.locked-unlocked").
running chrome
Error: An element could not be located on the page using the given search parameters ("[data-instrument_id="undefined"] input.locked-unlocked").
    at Context.it (test/instrument.js:97:16)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("[data-instrument_id="undefined"] input.locked-unlocked") - getValue.js:18:17
    at getValue("[data-instrument_id="undefined"] input.locked-unlocked") - at Object.me.toggleLockFromList (test/lib/instrument.js:54:30)

20) instrument should open instrument editor from list:
An element could not be located on the page using the given search parameters.
running chrome
Error: An element could not be located on the page using the given search parameters.
    at Context.it (test/instrument.js:100:16)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at element("[data-instrument_id="undefined"] a.pvedit") - click.js:12:17
    at click("[data-instrument_id="undefined"] a.pvedit") - at Object.me.gotoEditFromList (test/lib/instrument.js:67:19)

21) instrument should open the section editor:
An element could not be located on the page using the given search parameters.
running chrome
Error: An element could not be located on the page using the given search parameters.
    at Context.it (test/instrument.js:103:13)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at element("div.simLink[title="Edit Section Sort Order"]") - click.js:12:17
    at click("div.simLink[title="Edit Section Sort Order"]") - at Object.me.openSectionEditor (test/lib/section.js:43:12)

22) instrument should create a section:
An element could not be located on the page using the given search parameters ("#label").
running chrome
Error: An element could not be located on the page using the given search parameters ("#label").
    at node_modules/lodash/lodash.js:4925:15
    at baseForOwn (node_modules/lodash/lodash.js:3010:24)
    at node_modules/lodash/lodash.js:4894:18
    at Function.forEach (node_modules/lodash/lodash.js:9342:14)
    at Object.me.create (test/lib/section.js:50:7)
    at Context.it (test/instrument.js:114:13)
    at elements("#label") - setValue.js:29:17
    at setValue("#label", "selenium 1q/page section label") - at _.forEach (test/lib/section.js:52:27)

23) instrument should create another section:
An element could not be located on the page using the given search parameters ("#label").
running chrome
Error: An element could not be located on the page using the given search parameters ("#label").
    at node_modules/lodash/lodash.js:4925:15
    at baseForOwn (node_modules/lodash/lodash.js:3010:24)
    at node_modules/lodash/lodash.js:4894:18
    at Function.forEach (node_modules/lodash/lodash.js:9342:14)
    at Object.me.create (test/lib/section.js:50:7)
    at Context.it (test/instrument.js:125:13)
    at elements("#label") - setValue.js:29:17
    at setValue("#label", "selenium 3q/page section label") - at _.forEach (test/lib/section.js:52:27)

24) instrument should create a likert section:
An element could not be located on the page using the given search parameters ("#label").
running chrome
Error: An element could not be located on the page using the given search parameters ("#label").
    at node_modules/lodash/lodash.js:4925:15
    at baseForOwn (node_modules/lodash/lodash.js:3010:24)
    at node_modules/lodash/lodash.js:4894:18
    at Function.forEach (node_modules/lodash/lodash.js:9342:14)
    at Object.me.create (test/lib/section.js:50:7)
    at Context.it (test/instrument.js:137:13)
    at elements("#label") - setValue.js:29:17
    at setValue("#label", "selenium 3q/page Likert section label") - at _.forEach (test/lib/section.js:52:27)

25) instrument should create a table section:
An element could not be located on the page using the given search parameters ("#label").
running chrome
Error: An element could not be located on the page using the given search parameters ("#label").
    at node_modules/lodash/lodash.js:4925:15
    at baseForOwn (node_modules/lodash/lodash.js:3010:24)
    at node_modules/lodash/lodash.js:4894:18
    at Function.forEach (node_modules/lodash/lodash.js:9342:14)
    at Object.me.create (test/lib/section.js:50:7)
    at Context.it (test/instrument.js:149:13)
    at elements("#label") - setValue.js:29:17
    at setValue("#label", "selenium 4q/page TABLE section label") - at _.forEach (test/lib/section.js:52:27)

26) instrument should create a side by side section:
An element could not be located on the page using the given search parameters ("#label").
running chrome
Error: An element could not be located on the page using the given search parameters ("#label").
    at node_modules/lodash/lodash.js:4925:15
    at baseForOwn (node_modules/lodash/lodash.js:3010:24)
    at node_modules/lodash/lodash.js:4894:18
    at Function.forEach (node_modules/lodash/lodash.js:9342:14)
    at Object.me.create (test/lib/section.js:50:7)
    at Context.it (test/instrument.js:161:13)
    at elements("#label") - setValue.js:29:17
    at setValue("#label", "selenium 4q/page SXS section label") - at _.forEach (test/lib/section.js:52:27)

27) instrument should create a sacraficial section:
An element could not be located on the page using the given search parameters ("#label").
running chrome
Error: An element could not be located on the page using the given search parameters ("#label").
    at node_modules/lodash/lodash.js:4925:15
    at baseForOwn (node_modules/lodash/lodash.js:3010:24)
    at node_modules/lodash/lodash.js:4894:18
    at Function.forEach (node_modules/lodash/lodash.js:9342:14)
    at Object.me.create (test/lib/section.js:50:7)
    at Context.it (test/instrument.js:172:13)
    at elements("#label") - setValue.js:29:17
    at setValue("#label", "selenium 1q/page sacraficial section label") - at _.forEach (test/lib/section.js:52:27)

28) instrument should create a sacraficial section with questions:
An element could not be located on the page using the given search parameters ("#label").
running chrome
Error: An element could not be located on the page using the given search parameters ("#label").
    at node_modules/lodash/lodash.js:4925:15
    at baseForOwn (node_modules/lodash/lodash.js:3010:24)
    at node_modules/lodash/lodash.js:4894:18
    at Function.forEach (node_modules/lodash/lodash.js:9342:14)
    at Object.me.create (test/lib/section.js:50:7)
    at Context.it (test/instrument.js:183:13)
    at elements("#label") - setValue.js:29:17
    at setValue("#label", "selenium 1q/page sacraficial section with questions label") - at _.forEach (test/lib/section.js:52:27)

29) instrument should close the section editor:
An element could not be located on the page using the given search parameters.
running chrome
Error: An element could not be located on the page using the given search parameters.
    at Context.it (test/instrument.js:186:13)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at element(".tclose") - click.js:12:17
    at click(".tclose") - at Object.me.closeSectionEditor (test/lib/section.js:62:40)

30) instrument should open the question creation form:
An element could not be located on the page using the given search parameters ("#asmtAddQuestion").
running chrome
Error: An element could not be located on the page using the given search parameters ("#asmtAddQuestion").
    at Context.it (test/instrument.js:189:14)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.openQuestionCreator (test/lib/question.js:54:8)

31) instrument should add a question to first section:
An element could not be located on the page using the given search parameters ("[name=question_ID]").
running chrome
Error: An element could not be located on the page using the given search parameters ("[name=question_ID]").
    at node_modules/lodash/lodash.js:4925:15
    at baseForOwn (node_modules/lodash/lodash.js:3010:24)
    at node_modules/lodash/lodash.js:4894:18
    at Function.forEach (node_modules/lodash/lodash.js:9342:14)
    at Object.me.create (test/lib/question.js:100:7)
    at Context.it (test/instrument.js:200:14)
    at elements("[name=question_ID]") - setValue.js:29:17
    at setValue("[name=question_ID]", "SELENIUMZn1akCx_01") - at _.forEach (test/lib/question.js:102:27)

32) instrument should duplicate the question:
An element could not be located on the page using the given search parameters ("#questionList_SELENIUMZn1akCx_01 .editQuestionButton").
running chrome
Error: An element could not be located on the page using the given search parameters ("#questionList_SELENIUMZn1akCx_01 .editQuestionButton").
    at Object.me.duplicate (test/lib/question.js:77:8)
    at Context.it (test/instrument.js:206:14)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.hoverEdit (test/lib/question.js:72:42)

33) instrument should delete a question:
An element could not be located on the page using the given search parameters ("#questionList_SELENIUMZn1akCx_02 .editQuestionButton").
running chrome
Error: An element could not be located on the page using the given search parameters ("#questionList_SELENIUMZn1akCx_02 .editQuestionButton").
    at Object.me.delete (test/lib/question.js:92:15)
    at Context.it (test/instrument.js:212:20)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.hoverEdit (test/lib/question.js:72:42)

34) instrument add multiple questions to first section should add more questions to the first section:
An element could not be located on the page using the given search parameters ("#asmtAddQuestion").
running chrome
Error: An element could not be located on the page using the given search parameters ("#asmtAddQuestion").
    at questions.forEach (test/instrument.js:238:18)
    at Array.forEach (native)
    at Context.it (test/instrument.js:237:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.openQuestionCreator (test/lib/question.js:54:8)

35) instrument add multiple questions to second section should navigate to the second section:
An element could not be located on the page using the given search parameters ("//*[@id="asmtPageNav"]").
running chrome
Error: An element could not be located on the page using the given search parameters ("//*[@id="asmtPageNav"]").
    at Context.it (test/instrument.js:310:18)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.gotoSection (test/lib/instrument.js:75:45)

36) instrument add multiple questions to second section should add more questions to the second section:
An element could not be located on the page using the given search parameters ("#asmtAddQuestion").
running chrome
Error: An element could not be located on the page using the given search parameters ("#asmtAddQuestion").
    at questions.forEach (test/instrument.js:314:18)
    at Array.forEach (native)
    at Context.it (test/instrument.js:313:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.openQuestionCreator (test/lib/question.js:54:8)

37) instrument add multiple questions to likert section should navigate to the third section:
An element could not be located on the page using the given search parameters ("//*[@id="asmtPageNav"]").
running chrome
Error: An element could not be located on the page using the given search parameters ("//*[@id="asmtPageNav"]").
    at Context.it (test/instrument.js:342:18)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.gotoSection (test/lib/instrument.js:75:45)

38) instrument add multiple questions to likert section should add questions to the third section:
An element could not be located on the page using the given search parameters ("#asmtAddQuestion").
running chrome
Error: An element could not be located on the page using the given search parameters ("#asmtAddQuestion").
    at questions.forEach (test/instrument.js:346:18)
    at Array.forEach (native)
    at Context.it (test/instrument.js:345:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.openQuestionCreator (test/lib/question.js:54:8)

39) instrument add multiple questions to likert section should duplicate the question:
An element could not be located on the page using the given search parameters ("#questionList_SELENIUMZn1akCx_10 .editQuestionButton").
running chrome
Error: An element could not be located on the page using the given search parameters ("#questionList_SELENIUMZn1akCx_10 .editQuestionButton").
    at Object.me.duplicate (test/lib/question.js:77:8)
    at Context.it (test/instrument.js:356:16)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.hoverEdit (test/lib/question.js:72:42)

40) instrument add multiple questions to likert section should duplicate the question:
An element could not be located on the page using the given search parameters ("#questionList_SELENIUMZn1akCx_10 .editQuestionButton").
running chrome
Error: An element could not be located on the page using the given search parameters ("#questionList_SELENIUMZn1akCx_10 .editQuestionButton").
    at Object.me.duplicate (test/lib/question.js:77:8)
    at Context.it (test/instrument.js:362:16)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.hoverEdit (test/lib/question.js:72:42)

41) instrument add multiple questions to likert section should duplicate the question:
An element could not be located on the page using the given search parameters ("#questionList_SELENIUMZn1akCx_10 .editQuestionButton").
running chrome
Error: An element could not be located on the page using the given search parameters ("#questionList_SELENIUMZn1akCx_10 .editQuestionButton").
    at Object.me.duplicate (test/lib/question.js:77:8)
    at Context.it (test/instrument.js:368:16)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.hoverEdit (test/lib/question.js:72:42)

42) instrument add questions to side by side section should navigate to the fifth section:
An element could not be located on the page using the given search parameters ("//*[@id="asmtPageNav"]").
running chrome
Error: An element could not be located on the page using the given search parameters ("//*[@id="asmtPageNav"]").
    at Context.it (test/instrument.js:426:18)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.gotoSection (test/lib/instrument.js:75:45)

43) instrument add questions to side by side section should add questions to the fifth section:
An element could not be located on the page using the given search parameters ("#asmtAddQuestion").
running chrome
Error: An element could not be located on the page using the given search parameters ("#asmtAddQuestion").
    at questions.forEach (test/instrument.js:430:18)
    at Array.forEach (native)
    at Context.it (test/instrument.js:429:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.openQuestionCreator (test/lib/question.js:54:8)

44) instrument add a question to the sacraficial section should navigate to the sacraficial section:
An element could not be located on the page using the given search parameters ("//*[@id="asmtPageNav"]").
running chrome
Error: An element could not be located on the page using the given search parameters ("//*[@id="asmtPageNav"]").
    at Context.it (test/instrument.js:451:18)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.gotoSection (test/lib/instrument.js:75:45)

45) instrument add a question to the sacraficial section should add questions to the sacraficial section:
An element could not be located on the page using the given search parameters ("#asmtAddQuestion").
running chrome
Error: An element could not be located on the page using the given search parameters ("#asmtAddQuestion").
    at questions.forEach (test/instrument.js:455:18)
    at Array.forEach (native)
    at Context.it (test/instrument.js:454:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.openQuestionCreator (test/lib/question.js:54:8)

46) navigate to asmt and fill out asmts asmt double entry without error should go to asmt and select NITEST (study_id 2319):
stale element reference: element is not attached to the page document
running chrome
Error: An element command failed because the referenced element is no longer attached to the DOM.
    at Object.me.selectAsmtStudy (test/lib/nav/navigation.js:54:8)
    at Context.it (test/asmt-double-entry.js:94:11)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elementIdClick("0.7238241194985748-4") - selectByAttribute.js:46:22

47) navigate to asmt and fill out asmts asmt double entry without error should select Calculation Test (instrument_id 26363):
An element could not be located on the page using the given search parameters.
running chrome
Error: An element could not be located on the page using the given search parameters.
    at Context.it (test/asmt-double-entry.js:108:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at element("#instrument_id") - selectByAttribute.js:22:17
    at selectByAttribute("#instrument_id", "value", 26363) - selectByValue.js:40:17
    at selectByValue("#instrument_id", 26363) - at Object.me.selectInstrument.instrumentId [as selectInstrument] (test/lib/dataEntry.js:8:6)

48) navigate to asmt and fill out asmts asmt double entry without error should fill out the first 4 values of the Data Entry: Cover Sheet form:
An element could not be located on the page using the given search parameters ("input[name=ursi]").
running chrome
Error: An element could not be located on the page using the given search parameters ("input[name=ursi]").
    at Context.it (test/asmt-double-entry.js:117:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("input[name=ursi]") - setValue.js:29:17
    at setValue("input[name=ursi]", "M06100119") - at Object.me.fillCoverSheetPart1.details [as fillCoverSheetPart1] (test/lib/dataEntry.js:12:6)

49) navigate to asmt and fill out asmts asmt double entry without error should fill out the remaining required inputs for Data Entry: Cover Sheet form:
An element could not be located on the page using the given search parameters ("input[name=assessment_starttime]").
running chrome
Error: An element could not be located on the page using the given search parameters ("input[name=assessment_starttime]").
    at Context.it (test/asmt-double-entry.js:122:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("input[name=assessment_starttime]") - setValue.js:29:17
    at setValue("input[name=assessment_starttime]", "00:14") - at Object.me.fillCoverSheetPart2.details [as fillCoverSheetPart2] (test/lib/dataEntry.js:20:6)

50) navigate to asmt and fill out asmts asmt double entry without error should partially fill in assessment, then save and escape:
An element could not be located on the page using the given search parameters ("textarea[name=asmt-Testcal_1-1]").
running chrome
Error: An element could not be located on the page using the given search parameters ("textarea[name=asmt-Testcal_1-1]").
    at Context.it (test/asmt-double-entry.js:127:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("textarea[name=asmt-Testcal_1-1]") - setValue.js:29:17
    at setValue("textarea[name=asmt-Testcal_1-1]", "1") - at Object.me.partiallyFillCalTestAsmt (test/lib/dataEntry.js:61:8)

51) navigate to asmt and fill out asmts asmt double entry without error should resume the assessment which was just escaped:
An element could not be located on the page using the given search parameters ("input[aria-controls=my_asmts]").
running chrome
Error: An element could not be located on the page using the given search parameters ("input[aria-controls=my_asmts]").
    at Context.it (test/asmt-double-entry.js:138:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("input[aria-controls=my_asmts]") - setValue.js:29:17
    at setValue("input[aria-controls=my_asmts]", "01-18-1924") - at Object.me.resumeEntry.asmtDate [as resumeEntry] (test/lib/dataEntry.js:90:6)

52) navigate to asmt and fill out asmts asmt double entry without error should enter the resumed assessment:
An element could not be located on the page using the given search parameters.
running chrome
Error: An element could not be located on the page using the given search parameters.
    at Context.it (test/asmt-double-entry.js:143:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at element("button[id=start_entry]") - click.js:12:17
    at click("button[id=start_entry]") - at Object.me.enterResumedEntry (test/lib/dataEntry.js:95:6)

53) navigate to asmt and fill out asmts asmt double entry without error should fill in assessment first entry:
An element could not be located on the page using the given search parameters ("textarea[name=asmt-Testcal_3-1]").
running chrome
Error: An element could not be located on the page using the given search parameters ("textarea[name=asmt-Testcal_3-1]").
    at Context.it (test/asmt-double-entry.js:148:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("textarea[name=asmt-Testcal_3-1]") - setValue.js:29:17
    at setValue("textarea[name=asmt-Testcal_3-1]", "3") - at Object.me.completePartialCalTestAsmt (test/lib/dataEntry.js:76:8)

54) navigate to asmt and fill out asmts asmt double entry without error should begin a new assessment:
An element could not be located on the page using the given search parameters ("button[id="new_asmt_bottom"]").
running chrome
Error: An element could not be located on the page using the given search parameters ("button[id="new_asmt_bottom"]").
    at Context.it (test/asmt-double-entry.js:152:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.beginNewAssessment (test/lib/dataEntry.js:49:26)

55) navigate to asmt and fill out asmts asmt double entry without error should select Calculation Test (instrument_id 26363) for the second time:
An element could not be located on the page using the given search parameters.
running chrome
Error: An element could not be located on the page using the given search parameters.
    at Context.it (test/asmt-double-entry.js:157:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at element("#instrument_id") - selectByAttribute.js:22:17
    at selectByAttribute("#instrument_id", "value", 26363) - selectByValue.js:40:17
    at selectByValue("#instrument_id", 26363) - at Object.me.selectInstrument.instrumentId [as selectInstrument] (test/lib/dataEntry.js:8:6)

56) navigate to asmt and fill out asmts asmt double entry without error should fill out the first 4 values of the Data Entry: Cover Sheet form:
An element could not be located on the page using the given search parameters ("input[name=ursi]").
running chrome
Error: An element could not be located on the page using the given search parameters ("input[name=ursi]").
    at Context.it (test/asmt-double-entry.js:163:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("input[name=ursi]") - setValue.js:29:17
    at setValue("input[name=ursi]", "M06100119") - at Object.me.fillCoverSheetPart1.details [as fillCoverSheetPart1] (test/lib/dataEntry.js:12:6)

57) navigate to asmt and fill out asmts asmt double entry without error should fill out the remaining required inputs for Data Entry: Cover Sheet form:
An element could not be located on the page using the given search parameters ("input[name=assessment_starttime]").
running chrome
Error: An element could not be located on the page using the given search parameters ("input[name=assessment_starttime]").
    at Context.it (test/asmt-double-entry.js:168:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("input[name=assessment_starttime]") - setValue.js:29:17
    at setValue("input[name=assessment_starttime]", "00:14") - at Object.me.fillCoverSheetPart2.details [as fillCoverSheetPart2] (test/lib/dataEntry.js:20:6)

58) navigate to asmt and fill out asmts asmt double entry without error should fill in assessment second entry:
An element could not be located on the page using the given search parameters ("textarea[name=asmt-Testcal_1-1]").
running chrome
Error: An element could not be located on the page using the given search parameters ("textarea[name=asmt-Testcal_1-1]").
    at Context.it (test/asmt-double-entry.js:173:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("textarea[name=asmt-Testcal_1-1]") - setValue.js:29:17
    at setValue("textarea[name=asmt-Testcal_1-1]", "1") - at Object.me.fillCalculationTestAsmt (test/lib/dataEntry.js:32:8)

59) navigate to asmt and fill out asmts asmt double entry without error should fill assessment search criteria and click search button:
An element could not be located on the page using the given search parameters ("input[name=ursi]").
running chrome
Error: An element could not be located on the page using the given search parameters ("input[name=ursi]").
    at Context.it (test/asmt-double-entry.js:182:14)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("input[name=ursi]") - setValue.js:29:17
    at setValue("input[name=ursi]", "M06100119") - at Object.me.fillAsmtSearchCriteria.details [as fillAsmtSearchCriteria] (test/lib/manage.js:8:6)

60) navigate to asmt and fill out asmts asmt double entry without error should select all assessments and download them:
An element could not be located on the page using the given search parameters.
running chrome
Error: An element could not be located on the page using the given search parameters.
    at Context.it (test/asmt-double-entry.js:195:14)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at element("input.select-all") - click.js:12:17
    at click("input.select-all") - at Object.me.downloadAsmt (test/lib/manage.js:59:6)

61) navigate to asmt and fill out asmts asmt double entry without error should check if the downloaded file exists:
Assessment download failed.
running chrome
Error: Assessment download failed.
    at Context.it (/Users/mstone/stuff/coins-browser-test/test/asmt-double-entry.js:208:15)
    at Promise.F (/Users/mstone/stuff/coins-browser-test/node_modules/core-js/library/modules/_export.js:36:28)

62) navigate to asmt and fill out asmts asmt double entry without error should select "responses" button in order to view assessment responses:
An element could not be located on the page using the given search parameters ("#asmt_grid>tbody>tr>td>a").
running chrome
Error: An element could not be located on the page using the given search parameters ("#asmt_grid>tbody>tr>td>a").
    at Context.it (test/asmt-double-entry.js:213:14)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.clickAsmtResponsesButton (test/lib/manage.js:24:6)

63) navigate to asmt and fill out asmts asmt double entry without error should find auto calc that matches: 10.760204081633:
expected -1 not to be -1 (false negative fail)
running chrome
AssertionError: expected -1 not to be -1 (false negative fail)
    at Assertion.fail (/Users/mstone/stuff/coins-browser-test/node_modules/should/cjs/should.js:275:17)
    at Assertion.value (/Users/mstone/stuff/coins-browser-test/node_modules/should/cjs/should.js:368:17)
    at Object.me.verifyAutoCalcResponseExists (/Users/mstone/stuff/coins-browser-test/test/lib/manage.js:34:18)
    at Context.it (/Users/mstone/stuff/coins-browser-test/test/asmt-double-entry.js:217:14)
    at Promise.F (/Users/mstone/stuff/coins-browser-test/node_modules/core-js/library/modules/_export.js:36:28)

64) navigate to asmt and fill out asmts asmt double entry with entry error and entry error correction should select Calculation Test (instrument_id 26363):
An element could not be located on the page using the given search parameters.
running chrome
Error: An element could not be located on the page using the given search parameters.
    at Context.it (test/asmt-double-entry.js:261:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at element("#instrument_id") - selectByAttribute.js:22:17
    at selectByAttribute("#instrument_id", "value", 26363) - selectByValue.js:40:17
    at selectByValue("#instrument_id", 26363) - at Object.me.selectInstrument.instrumentId [as selectInstrument] (test/lib/dataEntry.js:8:6)

65) navigate to asmt and fill out asmts asmt double entry with entry error and entry error correction should fill out the first 4 values of the Data Entry: Cover Sheet form:
An element could not be located on the page using the given search parameters ("input[name=ursi]").
running chrome
Error: An element could not be located on the page using the given search parameters ("input[name=ursi]").
    at Context.it (test/asmt-double-entry.js:266:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("input[name=ursi]") - setValue.js:29:17
    at setValue("input[name=ursi]", "M06100119") - at Object.me.fillCoverSheetPart1.details [as fillCoverSheetPart1] (test/lib/dataEntry.js:12:6)

66) navigate to asmt and fill out asmts asmt double entry with entry error and entry error correction should fill out the remaining required inputs for Data Entry: Cover Sheet form:
An element could not be located on the page using the given search parameters ("input[name=assessment_starttime]").
running chrome
Error: An element could not be located on the page using the given search parameters ("input[name=assessment_starttime]").
    at Context.it (test/asmt-double-entry.js:271:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("input[name=assessment_starttime]") - setValue.js:29:17
    at setValue("input[name=assessment_starttime]", "03:55") - at Object.me.fillCoverSheetPart2.details [as fillCoverSheetPart2] (test/lib/dataEntry.js:20:6)

67) navigate to asmt and fill out asmts asmt double entry with entry error and entry error correction should fill in assessment first entry:
An element could not be located on the page using the given search parameters ("textarea[name=asmt-Testcal_1-1]").
running chrome
Error: An element could not be located on the page using the given search parameters ("textarea[name=asmt-Testcal_1-1]").
    at Context.it (test/asmt-double-entry.js:276:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("textarea[name=asmt-Testcal_1-1]") - setValue.js:29:17
    at setValue("textarea[name=asmt-Testcal_1-1]", "1") - at Object.me.fillCalculationTestAsmt (test/lib/dataEntry.js:32:8)

68) navigate to asmt and fill out asmts asmt double entry with entry error and entry error correction should begin a new assessment:
An element could not be located on the page using the given search parameters ("button[id="new_asmt_bottom"]").
running chrome
Error: An element could not be located on the page using the given search parameters ("button[id="new_asmt_bottom"]").
    at Context.it (test/asmt-double-entry.js:280:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.beginNewAssessment (test/lib/dataEntry.js:49:26)

69) navigate to asmt and fill out asmts asmt double entry with entry error and entry error correction should select Calculation Test (instrument_id 26363) for the second time:
An element could not be located on the page using the given search parameters.
running chrome
Error: An element could not be located on the page using the given search parameters.
    at Context.it (test/asmt-double-entry.js:285:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at element("#instrument_id") - selectByAttribute.js:22:17
    at selectByAttribute("#instrument_id", "value", 26363) - selectByValue.js:40:17
    at selectByValue("#instrument_id", 26363) - at Object.me.selectInstrument.instrumentId [as selectInstrument] (test/lib/dataEntry.js:8:6)

70) navigate to asmt and fill out asmts asmt double entry with entry error and entry error correction should click the next button for the first section of the Data Entry: Cover Sheet form:
An element could not be located on the page using the given search parameters.
running chrome
Error: An element could not be located on the page using the given search parameters.
    at Context.it (test/asmt-double-entry.js:290:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at element("button[name=update]") - click.js:12:17
    at click("button[name=update]") - at Object.me.clickCoverSheetNextButton (test/lib/dataEntry.js:55:6)

71) navigate to asmt and fill out asmts asmt double entry with entry error and entry error correction should click the next button for the second section of the Data Entry: Cover Sheet form:
An element could not be located on the page using the given search parameters.
running chrome
Error: An element could not be located on the page using the given search parameters.
    at Context.it (test/asmt-double-entry.js:295:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at element("button[name=update]") - click.js:12:17
    at click("button[name=update]") - at Object.me.clickCoverSheetNextButton (test/lib/dataEntry.js:55:6)

72) navigate to asmt and fill out asmts asmt double entry with entry error and entry error correction should fill in assessment second entry:
An element could not be located on the page using the given search parameters ("textarea[name=asmt-Testcal_1-1]").
running chrome
Error: An element could not be located on the page using the given search parameters ("textarea[name=asmt-Testcal_1-1]").
    at Context.it (test/asmt-double-entry.js:300:17)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("textarea[name=asmt-Testcal_1-1]") - setValue.js:29:17
    at setValue("textarea[name=asmt-Testcal_1-1]", "1") - at Object.me.fillCalculationTestAsmt (test/lib/dataEntry.js:32:8)

73) navigate to asmt and fill out asmts asmt double entry with entry error and entry error correction should search for and find the asmt conflict:
An element could not be located on the page using the given search parameters ("input[type=search]").
running chrome
Error: An element could not be located on the page using the given search parameters ("input[type=search]").
    at Context.it (test/asmt-double-entry.js:309:14)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("input[type=search]") - setValue.js:29:17
    at setValue("input[type=search]", "M06100119 05/05/1946 v2") - at Object.me.findAsmtConflict (test/lib/manage.js:43:8)

74) navigate to asmt and fill out asmts asmt double entry with entry error and entry error correction should fix the error, then resolve the conflict:
An element could not be located on the page using the given search parameters.
running chrome
Error: An element could not be located on the page using the given search parameters.
    at Context.it (test/asmt-double-entry.js:313:14)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at element("input[value=">>>"]") - click.js:12:17
    at click("input[value=">>>"]") - at Object.me.fixAndResolveConflict (test/lib/manage.js:54:6)

75) navigate to asmt and fill out asmts asmt double entry with entry error and entry error correction should fill assessment search criteria and click search button:
An element could not be located on the page using the given search parameters ("input[name=ursi]").
running chrome
Error: An element could not be located on the page using the given search parameters ("input[name=ursi]").
    at Context.it (test/asmt-double-entry.js:322:14)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("input[name=ursi]") - setValue.js:29:17
    at setValue("input[name=ursi]", "M06100119") - at Object.me.fillAsmtSearchCriteria.details [as fillAsmtSearchCriteria] (test/lib/manage.js:8:6)

76) navigate to asmt and fill out asmts asmt double entry with entry error and entry error correction should select "responses" button in order to view assessment responses:
An element could not be located on the page using the given search parameters ("#asmt_grid>tbody>tr>td>a").
running chrome
Error: An element could not be located on the page using the given search parameters ("#asmt_grid>tbody>tr>td>a").
    at Context.it (test/asmt-double-entry.js:326:14)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Object.me.clickAsmtResponsesButton (test/lib/manage.js:24:6)

77) navigate to asmt and fill out asmts asmt double entry with entry error and entry error correction should find auto calc that matches: 10.760204081633:
expected -1 not to be -1 (false negative fail)
running chrome
AssertionError: expected -1 not to be -1 (false negative fail)
    at Assertion.fail (/Users/mstone/stuff/coins-browser-test/node_modules/should/cjs/should.js:275:17)
    at Assertion.value (/Users/mstone/stuff/coins-browser-test/node_modules/should/cjs/should.js:368:17)
    at Object.me.verifyAutoCalcResponseExists (/Users/mstone/stuff/coins-browser-test/test/lib/manage.js:34:18)
    at Context.it (/Users/mstone/stuff/coins-browser-test/test/asmt-double-entry.js:330:14)
    at Promise.F (/Users/mstone/stuff/coins-browser-test/node_modules/core-js/library/modules/_export.js:36:28)

78) Query Builder Demographic Data export via direct URSIs should set up:
An element could not be located on the page using the given search parameters.
running chrome
Error: An element could not be located on the page using the given search parameters.
    at Context.wrapper (node_modules/lodash/lodash.js:5213:19)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at element("#optListOfSubjects") - click.js:12:17
    at click("#optListOfSubjects") - at Context.setupQuery (test/query-builder.js:56:8)

79) Query Builder Demographic Data export via direct URSIs should setup demo data:
An element could not be located on the page using the given search parameters ("#optDemoDataOutput").
running chrome
Error: An element could not be located on the page using the given search parameters ("#optDemoDataOutput").
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at scroll() - at Context.setupDemograpicData (test/query-builder.js:149:6)

80) Query Builder Demographic Data export via direct URSIs should preview and export:
element ("#resultsarea") still without text after 10000ms
running chrome
Error: element ("#resultsarea") still without text after 10000ms
    at Context.previewAndExport (test/query-builder.js:251:6)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elementIdText("0.9477373085438496-2") - getText.js:35:50

81) Query Builder Assessment Data export via standard export should setup assessment data:
element ("#asmtDataDiv select[name=selInstrument] option[value="simple test"]") still not existing after 10000ms
running chrome
Error: element ("#asmtDataDiv select[name=selInstrument] option[value="simple test"]") still not existing after 10000ms
    at Context.setupAssessmentData (test/query-builder.js:173:6)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("#asmtDataDiv select[name=selInstrument] option[value="simple test"]") - isExisting.js:46:17
    at isExisting("#asmtDataDiv select[name=selInstrument] option[value="simple test"]") - waitForExist.js:67:22

82) Query Builder Assessment Data export via standard export should preview and export:
element ("#resultsarea") still without text after 10000ms
running chrome
Error: element ("#resultsarea") still without text after 10000ms
    at Context.previewAndExport (test/query-builder.js:251:6)
    at Context.wrapper (node_modules/lodash/lodash.js:5213:19)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elementIdText("0.88305463203684-10") - getText.js:35:50

83) Query Builder Assessment Data export via legacy export should set up:
element not visible: Element is not currently visible and may not be manipulated
running chrome
Error: An element command could not be completed because the element is not visible on the page.
    at Context.setupQuery (test/query-builder.js:105:8)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elementIdClick("0.88305463203684-4") - selectByVisibleText.js:46:22

84) Query Builder Assessment Data export via legacy export should setup assessment data:
Timeout of 30000ms exceeded. Try to reduce the run time or increase your timeout for test specs (http://webdriver.io/guide/testrunner/timeouts.html); if returning a Promise, ensure it resolves. (/Users/mstone/stuff/coins-browser-test/test/query-builder.js)
running chrome
Error: Timeout of 30000ms exceeded. Try to reduce the run time or increase your timeout for test specs (http://webdriver.io/guide/testrunner/timeouts.html); if returning a Promise, ensure it resolves. (/Users/mstone/stuff/coins-browser-test/test/query-builder.js)
    at Test.Runnable._timeoutError (/Users/mstone/stuff/coins-browser-test/node_modules/wdio-mocha-framework/node_modules/mocha/lib/runnable.js:440:10)
    at Timeout.<anonymous> (/Users/mstone/stuff/coins-browser-test/node_modules/wdio-mocha-framework/node_modules/mocha/lib/runnable.js:251:24)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)

85) Query Builder Assessment Data export via legacy export should preview and export:
Error waiting for condition
running chrome
Error: Error waiting for condition
    at Object.wait (/Users/mstone/stuff/coins-browser-test/node_modules/fibers/future.js:449:15)
    at Object.waitUntil (/Users/mstone/stuff/coins-browser-test/node_modules/wdio-sync/build/index.js:283:31)
    at Object.waitForCondition (/Users/mstone/stuff/coins-browser-test/util/pagination.js:34:8)
    at Object.<anonymous> (/Users/mstone/stuff/coins-browser-test/node_modules/vargs-callback/index.js:13:23)
    at /Users/mstone/stuff/coins-browser-test/node_modules/wdio-sync/build/index.js:210:29
    - - - - -
    at elementIdText("0.88305463203684-10") - getText.js:35:50

86) Query Builder Scan Data "before all" hook:
Promise was rejected with the following reason: unexpected alert open: {Alert text : }
running chrome
Error: Promise was rejected with the following reason: unexpected alert open: {Alert text : }
    at Context.goBack (test/query-builder.js:340:6)
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at elements("#optListOfSubjects") - isExisting.js:46:17
    at isExisting("#optListOfSubjects") - waitForExist.js:67:22

87) Query Builder CRUD "before all" hook:
unexpected alert open: {Alert text : }
running chrome
Error: A modal dialog was open, blocking this operation
    at Promise.F (node_modules/core-js/library/modules/_export.js:36:28)
    at back() - at Context.goBack (test/query-builder.js:339:6)


npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! coins-browser-test@1.0.0 start: `wdio wdio.conf.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the coins-browser-test@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mstone/.npm/_logs/2018-05-25T18_40_16_195Z-debug.log
```

muttavarapu commented:
There is an issue with getText function because of selenium which is causing subject-tags test to fail. I changed that to use customGetText function. I am not sure why the asmt navigation is failing for you. I have added a check for element exist before click which would hopefully make the test pass.
Except instrument.js, every other test should fail at exact same places as before.

mstone121 commented on wdio.conf.js:
```diff
@@ -0,0 +1,209 @@
+    'test/study-add.js',
+    'test/study-visit.js',
+    'test/study-details.js',
+    'test/logon.js',
```
I'm not sure if it's worth having tests like this one and `navigateToAsmt`. If we do keep them they should be moved to the top of the list since they test functionality present in other tests.

muttavarapu replied:
I will remove logon.js and navigateToAsmt.

muttavarapu commented on wdio.conf.js:
```diff
@@ -0,0 +1,209 @@
+    'test/fileImport.js',          // broken
+    'test/instrument.js',          // broken
+    'test/ocoins.js',              // broken
+    'test/asmt-double-entry.js',   // broken
```
You mean fileImport ? Yes that was not broken. We have to manually delete the instrument to get the test passed.

mstone121 replied:
I meant `asmt-double-entry.js`.

muttavarapu commented:
asmt-double-entry is broken for me before the version update. It fails in 4 of the last 6 steps.
I didn't get your comment. Is it still failing to select study even after my latest fix ?
It should wait for the step to complete as I added a wait for exist in selectAsmtStudy. We don't have a promise and in fact if we want to wrap that in a promise, we have to do a similar wait and resolve once the element exist.
So, right now the asmt-double-entry should fail at same places as before.

mstone121 commented:
That doesn't seem to work either. I think it's finding the element okay, but the page changes before the study_id can be set.

muttavarapu commented:
I have added a pause while navigating to ASMT and removed the tests the tests which are included in other tests.

mstone121 commented:
I needed a longer pause. 3000 ms worked.
